docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.profile

```diff
@@ -17,6 +17,15 @@ use Drupal\user\Entity\User;
 use Drupal\user\Entity\Role;
 use Drupal\user\RoleInterface;
 
+/**
+ * Implements hook_ckeditor_plugin_info_alter().
+ */
+function su_humsci_profile_ckeditor_plugin_info_alter(array &$definitions) {
+  if (isset($definitions['drupalfontawesome'])) {
+    $definitions['drupalfontawesome']['class'] = 'Drupal\su_humsci_profile\Plugin\CKEditorPlugin\HumsciFontAwesome';
```

Comment:
All this to alter a button? Damn. Be aware by doing this you (the project) are now responsible for keeping this class up to date.

Reply:
Yeah. its really kinda difficult and annoying. I hated having to do it. but without changing the icon, there are two flag icons. one for anchor link and another for font awesome. Maybe theres another way :man_shrugging:

Reply:
good idea. luckily i had recently worked on CKEditor api with the patch that is included in the PR